File: cinema-reservations-api/Model/CinemaHall.cs
using System.ComponentModel.DataAnnotations;

namespace cinema_reservations_api.Model {
    public class CinemaHall {
        public int CinemaHallId { get; set; }
        public int Seats { get; set; }
        public int Rows { get; set; }

        public override string ToString() {
            return $"{CinemaHallId} {Seats} {Rows}";
        }
    }
}

File: cinema-reservations-front/lib/model/cinema_hall.dart
import 'package:json_annotation/json_annotation.dart';

part 'cinema_hall.g.dart';

@JsonSerializable()
class CinemaHall {
  final int cinemaHallId;
  final int seats;
  final int rows;

  CinemaHall({
    required this.cinemaHallId,
    required this.seats,
    required this.rows,
  });

  factory CinemaHall.fromJson(Map<String, dynamic> json) => _$CinemaHallFromJson(json);

  Map<String, dynamic> toJson() => _$CinemaHallToJson(this);
}
